include/exadg/incompressible_navier_stokes/time_integration/time_int_bdf_dual_splitting.cpp

```diff
@@ -376,7 +380,13 @@ TimeIntBDFDualSplitting<dim, Number>::convective_step()
   if(this->print_solver_info() and not(this->is_test))
   {
     this->pcout << std::endl << "Explicit convective step:";
-    print_wall_time(this->pcout, timer.wall_time());
+    if(this->param.spatial_discretization == SpatialDiscretization::HDIV)
+    {
+      this->pcout << std::endl << "(mass system solved)";
```

Comment:
Why do we need this comment? Isn't this already reflected in the print function that prints iteration counts to screen? Instead, the above comment "Explicit convective step" has to be adjusted, because it is no longer explicit in case of HDIV (just "Convective step"). Note that this will cause tests to fail, because the output for the L2 case will change.